modules/API/external/external_api.js

```diff
@@ -366,6 +366,30 @@ export default class JitsiMeetExternalAPI extends EventEmitter {
         return iframe.contentWindow.document.getElementById('largeVideo');
     }
 
+    /**
+     * Getter for participant specific video element in Jitsi Meet.
+     *
+     * @param {string|undefined} participantId - Id of participant to return the video for.
+     * 
+     * @returns {HTMLElement|undefined} - The requested video. Will return the local video
+     * by default if participantId is undefined.
+     */
+    _getParticipantVideo(participantId) {
+        const iframe = this.getIFrame();
+
+        if (!iframe
+                || !iframe.contentWindow
+                || !iframe.contentWindow.document) {
+            return;
+        }
+
+        if (typeof participantId === 'undefined' || participantId === this._myUserID) {
+            return iframe.contentWindow.document.getElementById('localVideo_container');
+        }
+
+        return iframe.contentWindow.document.querySelector('#participant_${participantId} video');
```

Comment:
`'#participant_${participantId} video'` will not replace the `participantId`. I guess the intention was 
```
`#participant_${participantId} video`
```

Please test your code before pushing your changes!

Reply:
Thanks for bearing with me. I'm not used to the eslint workflow. Was so focused on getting rid of the eslint errors on commit that I introduced this new issue >.< Also I find it difficult to test the api because I don't have a jitsi meet server deployed.

Reply:
Thanks for fixing the issue!

Just FYI: All the iframe api changes should be pretty easy to test. I think you can even test locally with an html file wile that loads the compiled external_api.js and put 'meet.jit.si' as the domain to be loaded in the iframe. This way you won't need your own jitsi-meet deployment.

Reply:
Thanks! Will try that next time I work on external_api.js :)